### DISCUSSION TITLE
Multiselect select

### DISCUSSION THREAD

**Discussion by andredewaard** [chakra-ui/zag#674] [Ideas]
Are their any plans to support multi-select for the select component? or is this already doable with some kind of option/workaround i'm missing?

**Comment by Giamme** [comment]
You can extend the select component functionality to achieve the multiple selection feature. 

I did it for both the combobox and the select components, if you are using Vue3 I can share how I did it.

**Comment by segunadebayo** [comment]
Closing — the `select` machine now supports a `multiple` prop for multi-select behavior.
